### PR TITLE
151 global footer tweaks

### PR DESCRIPTION
**Pull request by stocker**
# READY

# Summary
- Update Global Footer component to visually match what is on stanford.edu.

# Needed By (Date)
- March 27, 2019

# Urgency
- include in Decanter 5.0 release

# Steps to Test

1. Pull this branch
1. Run grunt styleguide
2. Check that the global footer looks very similar at all breakpoints to stanford.edu site.  Test in all supported browsers.

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/151

### CODE REVIEW

**Comment by sherakama** [comment]
@stocker two formatting items to get to: https://codeclimate.com/github/SU-SWS/decanter/pull/293

File: core/scss/components/global-footer/_global-footer.scss [hunk 1]
```diff
@@ -12,7 +12,8 @@
 
 // Global styles for the global footer.
 .global-footer {
-  @include padding(2.5rem 10px);
+  @include modular-spacing(1,'padding-top');
```
Comment (yvonnetangsu) [inline core/scss/components/global-footer/_global-footer.scss]:
```suggestion
  @include modular-spacing(1, 'padding-top');
```

File: core/scss/components/global-footer/_global-footer.scss [hunk 2]
```diff
@@ -12,7 +12,8 @@
...
+  @include modular-spacing(1,'padding-bottom');
```
Comment (yvonnetangsu) [inline core/scss/components/global-footer/_global-footer.scss]:
```suggestion
  @include modular-spacing(1, 'padding-bottom');
```

**Comment by yvonnetangsu** [comment]
Btw, @sherakama @josephgknox , I sneaked in the a:focus/outline change (removing the outline:0) in with this PR - let us know if that's ok. If not, I'll take it out.

**Comment by josephgknox** [comment]
@yvonnetangsu totally OK! Thanks for doing that.

**Comment by josephgknox** [comment]
@stocker woot! Thanks for cleaning those up.

**Comment by sherakama** [comment]
Woot. Thanks all.

**Pull request merged**